CameraInspector/Frame.cpp

```diff
@@ -0,0 +1,46 @@
+#include "Frame.h"
+
+#include <opencv2\highgui\highgui.hpp>
+	
+namespace CameraInspector
+{
+
+Frame::Frame() : frame_(std::make_shared<cv::Mat>())
+{
+}
+
+Frame::Frame(int width, int height, void* data) : Frame::Frame()
```

Comment:
It make sense to align frame constructor to cv::Mat constructor
http://docs.opencv.org/3.1.0/d3/d63/classcv_1_1Mat.html#a51615ebf17a64c968df0bf49b4de6a3a